sw/airborne/modules/relative_localization_filter/discrete_ekf.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) Mario Coppola
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/relativelocalizationfilter/discreteekf.h"
+ * @author Mario Coppola
+ * Discrete Extended Kalman Filter for Relative Localization
+ */
+
+#ifndef DISCRETE_EKF_H
+#define DISCRETE_EKF_H
+
+// #include "fmatrix.h"
+#include "stdlib.h"
+#include "string.h"
+#include "math.h"
+
+#define EKF_N 7
+#define EKF_M 6
+
+typedef struct discrete_ekf {
```

Comment:
We don't use type defs in paparazzi. Please remove and use 'struct discrete_ekf'.